maven-plugin-plugin/pom.xml

```diff
@@ -270,8 +249,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
-        <version>3.6.0</version>
-        <!-- will use previous maven-plugin-plugin release to build current maven-plugin-plugin as configured in parent -->
+        <version>3.6.4</version>
```

Comment:
please also update version for reporting -> m-p-p

Reply:
and where do I do that?

Reply:
line 277

Reply:
tx, fixed